src/pages/[slug].tsx

```diff
@@ -31,13 +36,8 @@ export async function getServerSideProps(
       },
     };
   }
-  const config = fs.readFileSync(configPath, 'utf8');
```

Comment:
The slug page still has no translations. I need to investigate why, but VScode is making live hard. I need to fix my EOLs first.

Reply:
I fixed it. I'll push the fix soon